docs/book/src/reentrancy/index.md

```diff
@@ -45,3 +40,7 @@ To check if the current caller is a reentrant, you may call the `is_reentrant()`
 ```sway
 {{#include ../../../../examples/reentrancy/src/main.sw:is_reentrant}}
 ```
+
+## Cross Contract Reentrancy
+
+Cross-Contract Reentrancy is not possible on Fuel due to the use of Native Assets. As such, no contract calls are performed when assets are transferred. However standard security practices when relying on other contracts for state should still be applied, especially when making external calls.
```

Comment:
Isnt reentrancy a catchall term that includes contract calls too? I believe the contract call triggered from a payment on ethereum is just a special case that is particularly emphasized, but its not the only way for reentrancy to occur. Correct me if im wrong.

Reply:
Yes it occurs anytime a function in a contract is called again. This requires either a second call within the contract(intra-contract) or if an external contract call is made which then re-enters the contract again. This often happens on Ethereum when payments are made as code is executed when this happens, meaning the contract could be malicious. But since no external calls are made, only when deliberately calling a contract, no accidental external calls can occur.